test/cli/seed-make.spec.js

```diff
@@ -378,4 +379,75 @@ development: {
       );
     });
   });
+
+  describe('--timestamp-filename-prefix option: make seed with timestamp filename prefix', () => {
+    /**
+     * @type FileTestHelper
+     */
+    let fileHelper;
+    beforeEach(() => {
+      fileHelper = setupFileHelper();
+    });
+
+    afterEach(() => {
+      fileHelper.cleanup();
+    });
+
+    before(() => {
+      process.env.KNEX_PATH = '../knex.js';
```

Comment:
can you reset it to pre-test value in `after` statement?

Reply:
Ok